packages/flutter/lib/src/rendering/image.dart

```diff
@@ -424,4 +434,35 @@ class RenderImage extends RenderBox {
     properties.add(DiagnosticsProperty<bool>('invertColors', invertColors));
     properties.add(EnumProperty<FilterQuality>('filterQuality', filterQuality));
   }
+
+  @override
+  void clear() {
+    if (_selected ==  false)
+      return;
+    _selected = false;
+    markNeedsPaint();
+  }
+
+  @override
+  Future<ByteData?>? copy() {
+    if (_selected == false)
+      return null;
+    return _image?.toByteData(); // This is async :(
+  }
+
+  @override
+  bool update(Offset start, Offset end) {
+    final Rect localRect = Rect.fromPoints(globalToLocal(start), globalToLocal(end));
```

Comment:
This would definitely be useful, but i would be concerned about being too top down? I'm not quite sure. if we could basically hit test in a rect area that would solve this problem exactly